plugin/src/test/java/org/opensearch/ml/rest/RestMLRemoteInferenceIT.java

```diff
@@ -251,6 +252,7 @@ public void testOpenAIChatCompletionModel() throws IOException, InterruptedExcep
         assertNotNull(responseMap);
     }
 
+    @Ignore("text-davinci-edit-001 been deprecated on 2024-01-04 and replaced by /v1/chat/completions")
```

Comment:
why do we need to ignore this test? I updated here : https://github.com/opensearch-project/ml-commons/pull/1835/commits/5021cfd77c5f265815dafde17d94835ee5498c0b#diff-1551321199e798aacb50228526ec9e02603234147ac41c7ef228ff2e2954e323


But you can add this in your code as you have other changes also.

Reply:
I think this is duplicate with `testOpenAIChatCompletionModel` which use gpt-3.5 and the endpoint is same, do we need to have two test cases test same functionality?

Reply:
update to gpt-4. If we think keep just one is enough, we can remove it in future PR.